### ISSUE TITLE
Dailies all seem to have ticked over two days overnight, night of daylight saving switch

### ISSUE THREAD

**Issue by dburt** [HabitRPG/habitica#3231]
On Sunday we Eastern Australians changed our clocks away from daylight saving time (+11:00 --> +10:00) and it seems like there was an extra day in Habitica between when I went to bed Saturday and when I got up Sunday: I received damage from all dailies, and all streaks were reset :frowning:

The symptom seems similar to #1733 but that looks like a separate issue which was resolved.

My UID: f3794190-e40c-494d-a2c7-9818f610f93e

**Comment by deilann** [comment]
Was your browser open during this period, by chance?

**Comment by dburt** [comment]
I don't think it was. It usually isn't, but I can't be sure.

**Comment by crookedneighbor** [comment]
@dburt Sorry this happened to you. The site has gone through several code revisions since you opened this issue, so I'm going to close this issue. If you're still having trouble, please reopen it.

**Comment by dburt** [comment]
No worries. It's only once a year, even if it's still a problem, and I didn't notice it this year.
